sources/platform/actors/running/actor_standby.md

```diff
@@ -0,0 +1,73 @@
+it well. Please head to the Actor README to learn more about the capabilities of your chosen Actor.
+
+#### Is there any scaling to accommodate the incoming requests
+
+When you use the Actor in Standby mode, the system automatically scales the Actor to accommodate the incoming requests. Under the hood,
+the system starts new Actor runs, which you will see in the Actor runs tab, with the origin set to Standby.
+
+#### How do I customize Standby configuration
+
+The Standby configuration currently consists of the following properties:
+
+- **Max requests per run** - The maximum number of concurrent HTTP requests a single Standby Actor run can accept. If this limit is exceeded, the system starts a new Actor run to handle the request, which may take a few seconds.
+- **Desired requests per run** - The number of concurrent HTTP requests a single Standby Actor run is configured to handle. If this limit is exceeded, the system preemptively starts a new Actor run to handle the additional requests.
+- **Memory (MB)** - The amount of memory (RAM) allocated for the Actor in Standby mode, in megabytes. With more memory, the Actor can typically handle more requests in parallel, but this also increases the number of compute units consumed and the associated cost.
+- **Idle timeout (seconds)** - If a Standby Actor run doesn’t receive any HTTP requests within this time, the system will terminate the run. When a new request arrives, the system might need to start a new Standby Actor run to handle it, which can take a few seconds. A higher idle timeout improves responsiveness but increases costs, as the Actor remains active for a longer period.
+- **Build** - The Actor build that the runs of the Standby Actor will use. Can be either a build tag (e.g. `latest.`), or a build number (e.g. `0.1.2`).
+
+You can see these in the Standby tab of the Actor detail page. However, note that these properties are not configurable at the Actor level. If you wish to
+use the Actor-level hostname, this will always use the default configuration. To override this configuration, just create a new Task from the Actor.
+You can then head to the Standby tab of the created Task and modify the configuration as needed. Note that the task has a specific hostname, so make
+sure to use that in your application if you wish to use the custom configuration.
+
+#### Are the Standby runs billed differently
+
+No, the Standby runs are billed in the same fashion as the normal runs.
+However, running Actors in Standby mode might have unexpected costs, as the Actors run in the background and consume resources even when no requests are being sent until they are terminated after the idle timeout period.
+
+#### Are the Standby runs shared among users
+
+No, even if you use the Actor-level hostname with the default configuration, the background Actor runs for your requests are not shared with other users.
+
+#### How can I enable Standby for my Actor
```

Comment:
This part is completely missing some development instructions. Let's add them once we have the `Actor.config.standbyPort` part in SDK ready.

Reply:
I added example code